docs/core/ocap.md

```diff
@@ -39,7 +39,7 @@ foundation of an object capability system.
 >     to another object only through a preexisting chain of references.
 >     In short, "Only connectivity begets connectivity."
 
-For an introduction to object-capabilities, see [this article](https://en.wikipedia.org/wiki/Object-capability_model).
+For an introduction to object-capabilities, see this [Wikipedia article](https://en.wikipedia.org/wiki/Object-capability_model).
 
 ## Ocaps in practice
```

Comment:
@AdityaSripal can you update this document with the new capability module logic and the explanation you wrote in Slack? 🙏